### PR TITLE
feat(Recurring Job) UI support Recurring Job

### PR DESCRIPTION
**Pull request by smallteeths**
https://github.com/longhorn/longhorn/issues/467

Signed-off-by: Siye Wang <siye@rancher.com>

### CODE REVIEW

**Comment by innobead** [comment]
cc @c3y1huang

**Comment by innobead** [comment]
@c3y1huang gave some updates to me. This PR will get ready after some minor fixes later on and expect in the RC1.

**Comment by c3y1huang** [comment]
Tested build, LGTM.

File: src/services/volume.js [hunk 1]
```diff
@@ -119,3 +111,52 @@ export async function cancelExpansion(params) {
...
+
+export async function removeVolumeRecurringJob(params) {
+  return request({
+    url: params.url,
+    method: 'post',
+    data: {
+      name: params.name,
+      isGroup: params.isGroup,
+    },
+  })
```
Comment (innobead) [inline src/services/volume.js:144]: This is the same as the `recurringJobAdd`? I thought it should be delete action.

**Review by innobead — CHANGES_REQUESTED** [review]

File: src/services/volume.js (hunk 1 above)
Comment (smallteeths) [reply src/services/volume.js:144]: They call the url is actually not the same, delete call is selectedVolume.actions.recurringJobDelete, the parameters are the same. I can combine them but the names of the functions will be very strange.

I wanted to separate `delete` and `add` so I used these two functions.

**Review by innobead — APPROVED** [review]
LGTM

**Pull request merged**